orchestrator-salt/src/main/resources/salt-common/salt/fluent/template/databus_monitoring.conf.j2

```diff
@@ -23,14 +23,11 @@
     event_message_field              message
     headers                          {{ fluent.dbusMonitoringAppHeaders }}
     stream_name                      {{ fluent.dbusMonitoringStreamName }}
-    partition_key                    "#{Socket.gethostname}"
-    {%- if fluent.proxyUrl %}
-    proxy_url                        "{{ fluent.proxyUrl }}"
-    {%- if fluent.proxyAuth %}
+    partition_key                    "#{Socket.gethostname}"{%- if fluent.proxyUrl %}
+    proxy_url                        "{{ fluent.proxyUrl }}"{%- if fluent.proxyAuth %}
     proxy_username                   "{{ fluent.proxyUser }}"
-    proxy_password                   "{{ fluent.proxyPassword }}"
-    {% endif %}
-    {% endif %}
+    proxy_password                   "{{ fluent.proxyPassword }}"{% endif %}{% endif %}{% if fluent.noProxyHosts and fluent.fluentVersion > 3 %}
+    no_proxy                         "{{ fluent.noProxyHosts }}"{% endif %}
```

Comment:
should be inside the {%- if fluent.proxyUrl %} but after {%- if fluent.proxyAuth %} 's endif